CodeGenerator/Sources/CodeGenerator/Models/Waiter.swift

```diff
@@ -0,0 +1,106 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2017-2020 the Soto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Soto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+// used to decode waiters-2.json files
+struct Waiters: Decodable {
+    struct Waiter: Decodable {
+        enum State: String, Decodable {
```

Comment:
Does it make sense to pull this from core rather than redefining it? (Relates to the question about whether the enum will ever change)

Reply:
The CodeGenerator is not dependent on soto-core